### PR TITLE
fix: always use absolute positioning to prevent jumping Orbit slider

### PR DESCRIPTION
**Pull request by DanielRuf**
Before and after a slide is animted its position attribute is changed between `absolute` and `relative` which can produce a jump of the element after a slide change.

Closes https://github.com/zurb/foundation-sites/issues/11082

### CODE REVIEW

File: js/foundation.orbit.js [hunk 1]
```diff
@@ -344,11 +344,10 @@ class Orbit extends Plugin {
 
       if (this.options.useMUI && !this.$element.is(':hidden')) {
         Motion.animateIn(
-          $newSlide.addClass('is-active').css({'position': 'absolute', 'top': 0}),
+          $newSlide.addClass('is-active'),
           this.options[`animInFrom${dirIn}`],
           function(){
-            $newSlide.css({'position': 'relative', 'display': 'block'})
-            .attr('aria-live', 'polite');
+            $newSlide.attr('aria-live', 'polite');
```
Comment (ncoden) [inline js/foundation.orbit.js]: Slides are hidden when `motion-ui` is not used.

https://github.com/DanielRuf/foundation-sites/blob/dbdae96312bcccd0820d8ee43c088d5d6a1fab14/js/foundation.orbit.js#L153

We should keep `'display': 'block'`;

**Review by ncoden — CHANGES_REQUESTED** [review]

File: js/foundation.orbit.js (hunk 1 above)
Comment (DanielRuf) [reply js/foundation.orbit.js]: But this causes the jump then.

Comment (DanielRuf) [reply js/foundation.orbit.js]: Ah, just the block property?

Comment (ncoden) [reply js/foundation.orbit.js]: Yeah, and remove `'position': 'relative'` from line 153. But make some tests (with and without motionUI / `mui` classes), I'm not sure.

**Comment by ncoden** [comment]
@DanielRuf Tell me when you done your tests or if I should take care of them 😉

**Comment by DanielRuf** [comment]
> or if I should take care of them

Some help is really appreciated and would be great.

**Comment by ncoden** [comment]
> Yeah, and remove 'position': 'relative' from line 153. But make some tests (with and without motionUI / mui classes), I'm not sure.

I found a bug before figuring out that `'position': 'relative'` was not removed. I took care of this.

> **7daaba5 fix: remove position reset of Orbit slides preventing the "full-absolute" new behavior**
>
> All Orbit slides should always have an `absolute` positioning to prevent jumping Orbit slider (see #11107). The `'position': 'relative'` reset is no longer necessary here and break the Orbit behavior as inline styles overrides `.orbit-slide` class styles.

**Review by ncoden — APPROVED** [review]
> Yeah, and remove 'position': 'relative' from line 153. But make some tests (with and without motionUI / mui classes), I'm not sure.

I found a bug before figuring out that `'position': 'relative'` was not removed. I took care of this.

> **7daaba5 fix: remove position reset of Orbit slides preventing the "full-absolute" new behavior**
>
> All Orbit slides should always have an `absolute` positioning to prevent jumping Orbit slider (see #11107). The `'position': 'relative'` reset is no longer necessary here and break the Orbit behavior as inline styles overrides `.orbit-slide` class styles.

I made some tests and everything seems to work 👍

**Pull request merged**